server/lib/budget.js

```diff
@@ -634,8 +636,11 @@ export async function sumCollectivesTransactions(
       if (['netAmountInCollectiveCurrency', 'netAmountInHostCurrency'].includes(column)) {
         where[Op.and].push({
           [Op.or]: [
-            { type: DEBIT, [Op.not]: { kind: ['HOST_FEE', 'PAYMENT_PROCESSOR_FEE'], OrderId: { [Op.not]: null } } },
-            { type: CREDIT, kind: 'PAYMENT_PROCESSOR_COVER' },
+            {
+              type: DEBIT,
+              [Op.not]: { [Op.and]: { kind: ['HOST_FEE', 'PAYMENT_PROCESSOR_FEE'], OrderId: { [Op.not]: null } } },
```

Comment:
`[Op.and]` is already the default, right? 

```suggestion
              [Op.not]: { kind: ['HOST_FEE', 'PAYMENT_PROCESSOR_FEE'], OrderId: { [Op.not]: null } },
```

No requested change, just want to make sure we're on the same page for this line.

Reply:
It was necessary to get the parenthesis right in the condition.